src/index.js

```diff
@@ -1,2 +1,172 @@
+          localData[i].description = editinput.value;
+          localStorage.setItem('activities', JSON.stringify(localData));
+        }
+      }
+      editinput.parentElement.classList.remove('checkedcontainer');
+      todocontainer.replaceChild(todo, editinput);
+      todo.textContent = editinput.value;
+    }
+  });
+};
+
+const removetodo = (todo) => {
+  todoCon.removeChild(todo);
+  let count = 0;
+  const localData = JSON.parse(localStorage.getItem('activities'));
+  const data = Array.from(localData).filter((i) => i.completed === false);
+  data.map((i) => i.index = count += 1);
+  localStorage.setItem('activities', JSON.stringify(data));
+};
+
+textInput.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter' && textInput.value) {
+    addToList(textInput.value);
+    textInput.value = null;
+  }
+});
+
+const getFromLocal = () => {
+  const data = JSON.parse(localStorage.getItem('activities'));
+  data.map((i) => {
+    Array1.push(i);
+    const todocontainer = document.createElement('div');
+    todocontainer.className = 'todocontainer';
+    todocontainer.innerHTML += `
+      <input type="checkbox" class="check">
+      <span>${i.description}</span>
+      <i class="fa-solid fa-ellipsis-vertical menu"></i>
+      <i class="fa-solid fa-trash-can delete"></i>
+    `;
+    todoCon.appendChild(todocontainer);
+
+    const edit = document.querySelectorAll('.menu');
+    edit.forEach((i) => {
+      i.addEventListener('click', () => {
+        edittodo(todocontainer, i.previousElementSibling);
+        i.parentElement.classList.add('checkedcontainer');
+      });
+    });
+  });
+
+  const checkbox = document.querySelectorAll('.check');
+  checkbox.forEach((i) => {
+    i.addEventListener('click', () => {
+      i.parentElement.classList.toggle('checkedcontainer');
+      i.nextElementSibling.classList.toggle('inputvalue');
+      i.parentElement.lastElementChild.classList.toggle('trash');
+      i.parentElement.lastElementChild.previousElementSibling.classList.toggle('edit-disable');
+      updateLocal();
+    });
+  });
+
+  const removePng = document.querySelectorAll('.delete');
+  removePng.forEach((i) => {
+    i.addEventListener('click', () => {
+      removetodo(i.parentElement);
+    });
+  });
+
+  localStorage.setItem('activities', JSON.stringify(Array1));
+};
+
+window.addEventListener('load', getFromLocal);
+const updateLocal = () => {
+  const localData = JSON.parse(localStorage.getItem('activities'));
+  const todos = document.querySelectorAll('span');
+  for (let i = 0; i < todos.length; i++) {
+    if (todos[i].classList.contains('inputvalue')) {
+      localData[i].completed = true;
+    } else {
+      localData[i].completed = false;
+    }
+  }
+
+  localStorage.setItem('activities', JSON.stringify(localData));
+};
+
+const clear = () => {
+  const localData = JSON.parse(localStorage.getItem('activities'));
+  const todocontainer = document.querySelectorAll('.todocontainer');
```

Comment:
To keep things simple you could try having all your query selectors at the beginning of the file for easy readability and scope.